### PR TITLE
Add EDI Space

### PR DESCRIPTION
**Pull request by sambacha**
**Add space for ... Freight Trust Network $EDI** 

> we are renaming the network to 'as2.network' , 

Etherscan: https://etherscan.io/token/0x79C5a1Ae586322A07BfB60be36E1b31CE8C84A1e  
CoinGecko: https://www.coingecko.com/en/coins/freight-trust-network  
Website: https://freighttrust.com  
Twitter: https://twitter.com/freighttrustnet

\[as2.network\](https://as2.network)

\<3

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
You will want to add `logo.png` and `space.png` images.
https://docs.snapshot.page/guides/create-a-space#4-add-a-logo-and-space-images

**Comment by sambacha** [comment]
Fixed,  must have mistaken them as optional

File: spaces/edi/index.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "address": "0x79C5a1Ae586322A07BfB60be36E1b31CE8C84A1e",
+        "symbol": "EDI",
+         "decimals": 18,
+      }
+    },
+    {
+      "name": "erc20-balance-of",
+      "params": {
+        "address": "0x7D2D3688Df45Ce7C552E19c27e007673da9204B8",
+        "symbol": "evo[a-z,A-Z,0-9]",
```
Comment (bonustrack) [inline spaces/edi/index.json:21]: Can you confirm the address and symbol are correct? This seem to be aLEND token.

File: spaces/edi/index.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "token": "0x79C5a1Ae586322A07BfB60be36E1b31CE8C84A1e",
+  "name": "Freight Trust Network",
+  "network": "1",
+  "symbol": "EDI",
+  "skin": "edi",
```
Comment (bonustrack) [inline spaces/edi/index.json:6]: The skin should exist here: https://github.com/snapshot-labs/snapshot-spaces/tree/master/skins

**Review by gokaiorg — CHANGES_REQUESTED** [review]
We've made a major global update, please pull before commit again your changes.
You can use the documentation to help you.
https://docs.snapshot.page/guides/create-a-space#1-fork-snapshot-spaces-repository-here

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**